Toshi/Source/Toshi/Render/TRender.h

```diff
@@ -63,32 +60,32 @@ namespace Toshi
 		public TSingleton<TRenderInterface>
 	{
 
-		const int SYSRESOURCE_NUMOF = 12;
+		static constexpr int SYSRESOURCE_NUMOF = 12;
 
 	protected:
-		bool m_bInScene;		// 0x9
-		bool m_bCreated = false;		// 0xA
-		bool m_bDisplayCreated;	// 0xB
-		bool m_bIsEnabled;		// 0xD
+		bool m_bInScene;                  // 0x9
+		bool m_bCreated = false;          // 0xA
+		bool m_bDisplayCreated;           // 0xB
+		bool m_bIsEnabled;                // 0xD
 		TRenderContext* m_pRenderContext; // 0x18
-		void* m_aSysResources;		// 0x20
+		void* m_aSysResources;            // 0x20
 		//TNodeList
 
 	public:
-
 		//TRenderInterface() {}
 
-		inline TRenderContext* SetCurrentRenderContext(TRenderContext* a_pRenderContext) { m_pRenderContext = a_pRenderContext; return m_pRenderContext; }
-		inline TRenderContext* GetCurrentRenderContext() { return m_pRenderContext; }
+		bool CreateDisplay();
+		TRenderContext* CreateRenderContext();
+
+		TRenderContext* SetCurrentRenderContext(TRenderContext* a_pRenderContext) { m_pRenderContext = a_pRenderContext; return m_pRenderContext; }
+		TRenderContext* GetCurrentRenderContext() { return m_pRenderContext; }
 
 		virtual bool Create(); //{ return true; }
-		bool CreateDisplay();
-		inline bool IsInScene() { return m_bInScene; }
-		inline bool IsCreated() { return m_bCreated; }
```

Comment:
I found out that inline keyword is actually added automatically to any class function defined in header file

Reply:
Oh